### PR TITLE
Feat(apple): App hang documentation

### PR DESCRIPTION
**Pull request by brustolin**
Added documentation regarding App hanks detection.

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: src/platforms/apple/common/configuration/app-hangs.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: App Hangs
+sidebar_order: 11
+description: "Learning about App hangs report"
+---
+
+This integration tracks app hangs. This feature is available for iOS, tvOS, MacOS and watchOS.
```
Comment (imatwawana) [inline src/platforms/apple/common/configuration/app-hangs.mdx:7]:
```suggestion
This integration tracks app hangs. This feature is available for iOS, tvOS, MacOS, and watchOS.
```

File: src/platforms/apple/common/configuration/app-hangs.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+title: App Hangs
+sidebar_order: 11
+description: "Learning about App hangs report"
```
Comment (imatwawana) [inline src/platforms/apple/common/configuration/app-hangs.mdx:4]:
```suggestion
description: "Learn about how to add app hang detection reporting."
```

**Review by imatwawana — COMMENTED** [review]
Made some grammar/wording and style edits.

File: src/platforms/apple/common/configuration/app-hangs.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+---
+title: App Hangs
+sidebar_order: 11
+description: "Learn about how to add app hang detection reporting."
+---
+
+This integration tracks app hangs. This feature is available for iOS, tvOS, MacOS, and watchOS.
```
Comment (bruno-garcia) [inline src/platforms/apple/common/configuration/app-hangs.mdx:7]:
```suggestion
This integration tracks app hangs. This feature is available on iOS, tvOS, macOS, and watchOS.
```

**Review by bruno-garcia — APPROVED** [review]
One nit otherwise lgtm

**Pull request merged**